srcpkgs/dbeaver/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'dbeaver'
 pkgname=dbeaver
-version=7.3.2
+version=7.3.3
 revision=1
 # the build downloads binaries linked to glibc
 archs="x86_64"
 hostmakedepends="apache-maven"
+depends="openjdk11"
```

Comment:
This will have to change if we get new openjdk versions...

Reply:
Ah, so what's the right way to do this? Add a message to the installer and leave this to the user?

Reply:
I think it's ok to leave as depends for now. Just something to note for the future (maybe leave a comment?)